src/Interfaces/AbstractPolytope.jl

```diff
@@ -96,6 +97,60 @@ function isempty(P::AbstractPolytope)::Bool
     return isempty(vertices_list(P))
 end
 
+"""
+    chebyshev_center(P::AbstractPolytope{N};
+                     backend=default_polyhedra_backend(P, N),
+                     compute_radius::Bool=false) where {N<:Real}
+
+Compute the [Chebyshev center](https://en.wikipedia.org/wiki/Chebyshev_center)
+of a polytope.
+
+### Input
+
+- `P`              -- polytope
+- `backend`        -- (optional, default: `default_polyhedra_backend(P, N)`) the
+                      backend for polyhedral computations
+- `compute_radius` -- (optional; default: `false`) option to additionally return
+                      the radius of the largest ball enclosed by `P` around the
+                      Chebyshev center
+
+### Output
+
+If `compute_radius` is `false`, the result is the Chebyshev center of `P`.
+If `compute_radius` is `true`, the result is the pair `(c, r)` where `c` is the
+Chebyshev center of `P` and `r` is the radius of the largest ball with center
+`c` enclosed by `P`.
+
+### Notes
+
+The Chebyshev center is the center of a largest Euclidean ball enclosed by `P`.
+In general, the center of such a ball is not unique (but the radius is).
+
+Formally, letting ``∂P`` denote the boundary of ``P``, the Chebyshev center is
+
+```math
+    \\arg\\min_c \\min_{x ∈ ∂P} ‖x - c‖₂.
```

Comment:
[Wikipedia](https://en.wikipedia.org/wiki/Chebyshev_center) only lists the definition for the smallest enclosing ball and the book that they refer to also does not give a good definition. Since `Polyhedra` uses the interpretation "largest enclosed ball", I came up with this definition here. Please check it!

Reply:
i dont understand the formula in line 132; shouldn't there be a max? or if i take x = c any point in ∂P then the norm is zero => all points in the border are minimizers.

we can write it as `max r` subject to `B(c, r) ⊆ P,  c ∈ P`. the actual optimization problem implementing this is defined [here](https://github.com/JuliaPolyhedra/Polyhedra.jl/blob/737e49135c5bbfcd1e754398ec449d57e05aec2f/src/center.jl#L10) (which we could also add as math text here in the docstring).

Reply:
I will just remove the definition. The text should be sufficient.